site/gatsby-site/playwright/e2e-full/entityEdit.spec.ts

```diff
@@ -103,4 +103,73 @@ test.describe('Edit Entity', () => {
       await expect(page.locator('.tw-toast')).toContainText('Error updating Entity.');
     }
   );
+
+  test('Should successfully add Entity Relationship', async ({ page, login, skipOnEmptyEnvironment }) => {
+  
+    const userId = await login(process.env.E2E_ADMIN_USERNAME, process.env.E2E_ADMIN_PASSWORD);
+
+    await init();
+  
+    await init({ customData: { users: [{ userId, first_name: 'Test', last_name: 'User', roles: ['admin'] }] }, }, { drop: true });
```

Comment:
I updated the login fixture so it automatically adds the user to the db, you should only need to do 

```
init() // if necessary

login(...)

```

Reply:
(Look at existing examples once you merge latest staging)